e2e/runner/gov_propsoals.go

```diff
@@ -0,0 +1,186 @@
+package runner
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/gov/types"
+	govv1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+
+	"github.com/zeta-chain/node/e2e/txserver"
+)
+
+type ExecuteProposalSequence string
+
+const (
+	StartOfE2E ExecuteProposalSequence = "start"
+	EndOfE2E   ExecuteProposalSequence = "end"
```

Comment:
Can you explain in comment why having this option

Reply:
aa8582481b6baf9f146439ea2a882a625b0b6219